src/dss/create_notebook.py

```diff
@@ -78,51 +93,69 @@ def create_notebook(name: str, image: str, lightkube_client: Client) -> None:
 
     try:
         k8s_resource_handler.apply()
-
-        wait_for_deployment_ready(lightkube_client, namespace=DSS_NAMESPACE, deployment_name=name)
-
+        wait_for_deployment_ready(
+            lightkube_client, namespace=DSS_NAMESPACE, deployment_name=name, timeout_seconds=None
+        )
         logger.info(f"Success: Notebook {name} created successfully.")
+        if gpu:
+            logger.info(f"{gpu.title()} GPU attached to notebook.")
```

Comment:
Minor suggestion.  `title()` wont be correct in all cases (ex: `Amd`).  I'd stick to the enumerators we say `gpu` should be 
```suggestion
            logger.info(f"{gpu} GPU attached to notebook.")
```